bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -133,6 +157,28 @@
 		<item-type>Switch</item-type>
 		<label>Central Heating Overridden</label>
 		<description>Central heating enabled overridden at OTGW</description>
+		<state readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="ch2_enable">
+		<item-type>Switch</item-type>
+		<label>Central Heating Enabled</label>
```

Comment:
All `ch2_*` channel-type definitions miss `2` in the label and description.
```suggestion
		<label>Central Heating 2 Enabled</label>
```

Reply:
Added 2 to the labels and descriptions